### DISCUSSION TITLE
Getting Test Error with the sync PR

### DISCUSSION THREAD

**Discussion by arshadkazmi42** [reactjs/reactjs.org-translation#252] [Team Posts]
Hi,

I am maintaining the hindi translation.

Today there was a sync PR created from main repository.

One of the test is failing on the PR.

https://github.com/reactjs/hi.reactjs.org/pull/164#pullrequestreview-812138569

Anyone have any idea what could be the issue here?

**Comment by gaearon** [comment]
Sorry for no response — we're just picking work back up. If something doesn't work still, let us know.
